ci/run_ui_tests.sh

```diff
@@ -28,6 +28,6 @@ adb shell settings put global animator_duration_scale 0
 adb shell am start -n io.gnosis.safe.debug/io.gnosis.safe.ui.splash.SplashActivity
 
 echo "Execute UI tests"
-./gradlew createDebugTestCoverage --stacktrace
+./gradlew jacocoTestCoverageVerification --stacktrace
 
 bash <(curl -s https://codecov.io/bash) -f '*TestCoverage.xml'
```

Comment:
Why was this only done in the ui tests files? It seems jacocoTestReport has to run instrumentation tests since createDebugCoverageReport seems to depend on them. Should I try to make this run without instrumentation, or are we fine with running instrumntation on the ci. Apparently it worked in the past.